### DISCUSSION TITLE
Adressbook

### DISCUSSION THREAD

**Discussion by ednt** [rustdesk/rustdesk#5141] [General]
Is there a Documentation of the Adressbook for the Pro Server?

How can i link Devices to the Adressbook ok a colleague ?

**Comment by rustdesk** [comment]
The addressbook is linked to one account, not designed to be shared between accounts.

**Comment by rustdesk** [comment]
We are now refactoring the addressbook also, allow sync with recent sessions (including password), or maybe also sync with assigned devices under the login account.

**Comment by ednt** [comment]
It would be nice to sync the Address book inside the Groups

So all users are inside the group can see these accounts.

Also, a Public Book would make sense.

**Reply by rustdesk** [reply]
Make sense, we will do this, may be in 1.1.10 or later version.

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk/status/1777180459429986787
